packages/website/ts/components/portal/portal.tsx

```diff
@@ -155,6 +154,12 @@ export class Portal extends React.Component<PortalProps, PortalState> {
             // tslint:disable-next-line:no-floating-promises
             this._fetchBalancesAndAllowancesAsync(this._getCurrentTrackedTokensAddresses());
         }
+        if (!prevProps.isPortalOnboardingShowing && this.props.isPortalOnboardingShowing) {
+            // On mobile, make sure the wallet is completely visible.
+            if (this.props.screenWidth === ScreenWidths.Sm) {
+                document.querySelector('.wallet').scrollIntoView();
+            }
+        }
```

Comment:
This is a good general solution to this problem actually -- want to remove the other references to `scrollIntoView` and just stick this in `didComponentUpdate` of `PortalOnboardingFlow` ?

Reply:
Hm, I only see this one place `scrollIntoView` is being used, perhaps the other references are in your polish branch

Reply:
that is correct.